### DISCUSSION TITLE
SQLite rather than SQLite3

### DISCUSSION THREAD

**Discussion by Modcat** [feathersjs/feathers#2516] [Ideas]
I think it's best to work with the main SQLite package as SQLite3 doesn't compile in the latest electron for Linux as the binaries are missing for electron forge. SQLite is now on version 4 should be just as compatible and I think we should use SQLite now on version 4 than sticking with old technology.

**Comment by daffl** [comment]
That makes sense if Sequelize and Knex work with it. I kept seeing issues with sqlite3 it but didn't know there was a different package available.
